### ISSUE TITLE
[question] Color of signatures and classes for C++ generated with breathe are all in the same color

### ISSUE THREAD

**Issue by maximiliank** [readthedocs/sphinx_rtd_theme#1212]
### Problem

The generated documentation for classes and signatures are all in the same color and therefore it is harder to read.
I would expect that `MyClass` appears in black (similar for the function signature) and not all in the same color:

![example](https://user-images.githubusercontent.com/9783517/130076764-07da761d-4533-47e4-a689-9a663288d771.png)

Is it possible to configure the colors for keywords? If I look at examples in previous versions it was possible to get `MyClass` in black.

### Reproducible Project

`cat Source/myclass.h`:
```cpp
/**
 * @brief A class
 *
 */
class MyClass
{
public:
  /**
   * @brief Function that squares its argument
   *
   * @param a An integer parameter
   * @return the squared value of a, i.e. a*a
   */
  int square(int a) const;
}
```
`cat Doxyfile`:
```
PROJECT_NAME           = "testdoc"
GENERATE_HTML          = NO
GENERATE_LATEX         = NO
GENERATE_XML           = YES
INPUT                  = "Source"
OUTPUT_DIRECTORY       = "doxygen"
```
`cat conf.py`:
```python
project = 'Test Documentation'
extensions = ['breathe']
html_theme = 'sphinx_rtd_theme'
breathe_projects = {  "testdoc": "doxygen/xml/" }
```
`cat index.rst`:
```rst
.. doxygenclass:: MyClass 
   :project: testdoc
   :members:
   :undoc-members:
```

Run it with:
```shell
doxygen Doxyfile
sphinx-build -b html . sphinx_html
```

#### Expected Results

I would expect syntax highlighting for class definitions and signatures.
 
### Environment Info

- Python Version: 3.9.5
- Sphinx Version: 4.1.2
- RTD Theme Version: 0.5.2
- Breathe: 4.30.0

**Comment by dp0s** [comment]
Interestingly, I had the same problem for RTD Theme Version: 0.5.2 (see pictures in my issue mentioned above), but when updating to 1.0.0rc1 the colors turned back to "normal" for my local build.

**Comment by maximiliank** [comment]
Right, I accidentally still used the old system installation and not the 1.0.0rc1 installed via pip. So I can confirm, the color problem is fixed with 1.0.0rc1. Thank you.
